core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/ExpansionGreedyScheduleGenerator.scala

```diff
@@ -202,6 +212,9 @@ class ExpansionGreedyScheduleGenerator(
     // mark links that go to downstream regions
     populateDependeeLinks(regionDAG)
 
+    // add both materialized output ports and view-result output ports to regions
```

Comment:
I suggest removing this comment. in the engine, we don't differentiate the cause of materializations.

Reply:
Removed.